prow/plank/controller.go

```diff
@@ -218,6 +218,7 @@ func (c *Controller) Sync() error {
 
 	var reportErrs []error
 	if c.ghc != nil {
+		c.log.Warn("****** REPORTING ******")
```

Comment:
we shouldn't merge with this line included? feel free to test though. I didn't check the rest of the code yet but I think we just needed to fix `NewController`

Reply:
the commit message is also stale now

Reply:
```
{"component":"plank","level":"warning","msg":"****** REPORTING ******","time":"2018-07-03T01:07:58Z"}
```
nope

I think you are not suppose to compare an interface directly with nil

Reply:
https://play.golang.org/p/Isoo0CcAvr

Reply:
https://play.golang.org/p/O0bzko4Mg66